contents/handbook/people/onboarding.md

```diff
@@ -86,6 +86,40 @@ Add these if appropriate for the role:
 <input type="checkbox"/>  Hubspot if they are involved in customer-facing roles (e.g. sales, user interviews) <br>
 <input type="checkbox"/>  Any relevant banking or accounting software (very unlikely) <br>
 
+## Tools we use
+
+We use a number of different tools to organise our work and communicate at PostHog. Below is a summary list of the most important ones - this list is not intended to be exhaustive
+
+### Everyone
+- Google Suite - Gmail, Google Apps such as Docs, Sheets, Slides 
+- GitHub - most comms and product work
+- Slack - we have an internal workspace and a users Slack as well
+- Brex (US) or Revolut (UK) - company cards
+- Expensify - team expenses
+- Shopify - powers our merch store
+- CharlieHR - holiday tracking, personal details
+
+### Engineering
+- AWS
+- Pagerduty
+- Heroku
+- Grafana
+- Sentry
+
+### Design
+- Figma - our main design tool
+
+### Ops
+- HubSpot - for managing all sales
+- Papercups - our support platform
+- Pry - financial modelling
+- Pulley - cap table management
```

Comment:
Aren't we moving to captable?

Reply:
We are, but for now we still use Pulley. Once I've got everything migrated to captable (they are still setting up for us) then I was planning to update this.